Mudd/MuddCommTest.cpp

```diff
@@ -58,7 +58,8 @@ void StartClient()
         {
             if (line != "")
             {
-                msgs.Push(CommandDecoder::Decode(line));
+                auto msg = CommandDecoder::Decode(line);
```

Comment:
couldn't pass a temporary unique_ptr into a function taking a unique_ptr&.  
So I just made a local copy before passing it in.  Only way I could get it to compile